barretenberg/cpp/src/barretenberg/circuit_checker/ultra_circuit_builder.test.cpp

```diff
@@ -646,7 +646,77 @@ TEST(ultra_circuit_constructor, non_native_field_multiplication)
     EXPECT_EQ(result, true);
 }
 
-TEST(ultra_circuit_constructor, rom)
+/**
```

Comment:
this test copies the previous test and uses the function that previously, but now doesn't, added a gate with q_arith turned on in the aux block. It checks that the aux block does not have any other selector turned on besides the aux one

Reply:
Would be nice to have this, but you're only checking one block.